src/applications/personalization/preferences/components/PreferenceItem.jsx

```diff
@@ -52,57 +52,81 @@ const FAQList = ({ faqs }) => (
   </div>
 );
 
-export default function PreferenceItem({
-  handleViewToggle,
-  handleRemove,
-  isRemoving,
-  benefit,
-}) {
-  const { title, introduction, code, cta, faqs } = benefit;
+export default class PreferenceItem extends React.Component {
+  constructor(props) {
+    super(props);
 
-  if (isRemoving) {
+    this.removeBtnRef = React.createRef();
+    this.alertHeaderRef = React.createRef();
+    this.removeCancelled = false;
+  }
+
+  componentDidUpdate() {
+    // Set focus for keyboard-users upon content-swap.
+    if (this.props.isRemoving) {
+      this.alertHeaderRef.current.focus();
+    } else if (this.removeCancelled) {
+      this.removeBtnRef.current.focus();
+      this.removeCancelled = false;
```

Comment:
I missed this on my first review, but should `removeCancelled` be in state? Or can we derive this value by comparing `this.props.isRemoving` to `prevProps.isRemoving`? I haven't worked with this component so I don't know, but just wondering.

Reply:
@ncksllvn, I just noticed you Approved this PR, but I can definitely make this change if you like.
Just let me know.  Thx much!

Reply:
Completely up to you. This looks great me!

Reply:
Refactored.  Removed that unneeded flag and using prevProps compare instead.

Reply:
Well done! Looks great.